experimental/PropertyDDS/packages/property-binder/src/data_binder/dataBinder.ts

```diff
@@ -456,7 +456,7 @@ export class DataBinder {
 		in_definition: DataBindingDefinition,
 	) {
 		const byBindingType = this._definitionsByBindingType.get(in_definition.bindingType);
-		console.assert(byBindingType);
+		console.assert(byBindingType !== undefined);
```

Comment:
A side effect of removing the node module resolution assertion is that `console.assert` is now type checked against the Browser's console rather than Nodes.

Browsers say the first argument must be boolean | undefined.